subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/horreum_lts_store.py

```diff
@@ -161,27 +161,27 @@ def build_lts_payloads() -> dict:
 
         start_time: datetime.datetime = results.start_time
         end_time: datetime.datetime = results.end_time
-
-        output = {
+        payload = {
             "$schema": "urn:rhods-matbench-upload:3.0.0",
             "data": {
                 "users": _decode_users(results),
                 'rhods_version': results.rhods_info.version,
                 'ocp_version': results.sutest_ocp_version,
                 'metrics': _gather_prom_metrics(entry),
                 'thresholds': results.thresholds,
-                'config': results.test_config.yaml_file
+                'config': results.test_config.yaml_file,
+                "cluster_info": _parse_entry(entry.results.rhods_cluster_info),
             },
             "metadata": {
-                "test": "rhods-notebooks",
+                "test": results.test_config.yaml_file['tests']['identifier'],  
```

Comment:
```suggestion
                "test": results.test_config.get("tests.identifier"),
```
what happens on Horreum side if this field is empty?